nexus/src/app/background/common.rs

```diff
@@ -281,6 +281,7 @@ impl Driver {
     }
 
     /// Returns the runtime status of the background task
+    #[allow(dead_code)]
     pub fn status(&self, task: &TaskHandle) -> TaskStatus {
```

Comment:
It's being used by unit tests right now, though it looks "kinda reasonable to use from other contexts".

I marked it as `allow(dead_code)` rather than `cfg(test)` for that reason -- I think it would be reasonable for other parts of Nexus to act on this status, even though only tests are doing so right now.